packages/core/admin/_internal/node/develop.ts

```diff
@@ -104,24 +148,44 @@ const develop = async ({
   }
 
   if (cluster.isWorker) {
+    const strapi = strapiFactory({
+      appDir: cwd,
+      distDir: tsconfig?.config.options.outDir ?? '',
+      autoReload: true,
+      serveAdminPanel: !watchAdmin,
+    });
+
+    const strapiInstance = await strapi.load();
```

Comment:
can you still add middlewares to a loaded strapi app?

Reply:
We load strapi so that it gets all the user code loaded into it and can pick up any type modifications in it, is that what you mean or am I missing something?

Reply:
further down the bundlers inject server middlewares e.g. `webpack-dev-middleware` so I'm wondering if these still are injected & work?

Reply:
`watchWebpack` for example fyi

Reply:
Ahhh, good catch, I see. I probably need to move that up next to the load (and hope I don't have issues in doing so). I'll ping you when it's done, probably tomorrow

Reply:
Nm, it was simple, all good now I think.